app/models/refusal_advice.rb

```diff
@@ -40,6 +40,29 @@ def snippets
     scope.without_tag('internal_review')
   end
 
+  ##
+  # Retrieve the previous answers provided by the refusal advice wizard for a
+  # given InfoRequest and user.
+  #
+  def answers
```

Comment:
How about `suggested_actions`? On first read I thought this was more to do with the answers given by the user.

This reads a little nicer to me:

```ruby
# Humanised reading: "The refusal advice suggests you challenge the aggregation periods"
@refusal_advice.suggested_actions
# => ["refusal_advice:action-challenge-aggregation-periods"]
```